### DISCUSSION TITLE
Markdown support?

### DISCUSSION THREAD

**Discussion by sandeep-datta** [freeplane/freeplane#354] [General]
Hi, it seems markdown support was added and later [removed](https://github.com/freeplane/freeplane/commit/1b14b37d3a184117031ea877d9381a7ccde688bc) but I can't seem to find any discussion on why this was done. Is markdown support still in the works?

**Comment by macmarrum** [comment]
Hi @sandeep-datta 

Markdown is supported since 1.9.1. 

Since 1.9.12, markdown or HTML content can be styled using CSS – please see https://freeplane.github.io/wiki/#/css-for-html-or-markdown

**Reply by sandeep-datta** [reply]
That is fantastic, thanks!
